### ISSUE TITLE
2019 dev-meeting in Berlin

### ISSUE THREAD

**Issue by jakob-wo** [oemof/organisation#46]
I would like to invite you to Berlin for the next oemof developer meeting in winter 2019!
The venue is the _Beuth Hochschule für Technik Berlin_. As nice conference rooms are scarce we would like to fix the date as soon as possible and an early fixing hopefully makes the participation more feasable for everyone.
There is a conference room available in the beginning of December. So I suggest the following date:
From **Wednesday the 4th until Friday the 6th of December 2019**.

Is the date convenient for you @oemof-developer ?

**Comment by uvchik** [comment]
I do not have anything else, so I would attend. Thank you for the invitation :smile:

**Comment by p-snft** [comment]
The week before, there is the "Helmholtz-Sustainability-Summit", also in Berlin. For those of us working at Helmholtz institutes (Is it just me?), it could be possible to combine the two dates.

**Comment by c-moeller** [comment]
For me it's also possible to attend.

**Comment by FranziPl** [comment]
I would attend, too.

**Comment by oemof-developer** [comment]
This is actually the only date I am available in December, so I am in! 😉

Gesendet von IBM Verse


   FranziPl --- Re: [Oemof] [oemof/organisation] 2019 dev-meeting in Berlin (#46) --- 
    Von:"FranziPl" <notifications@github.com>An:"oemof/organisation" <organisation@noreply.github.com>Kopie:"oemof-developer" <dev@oemof.org>, "Mention" <mention@noreply.github.com>Datum:Do. 06.06.2019 10:54Betreff:Re: [Oemof] [oemof/organisation] 2019 dev-meeting in Berlin (#46)
  
 I would attend, too. —

**Comment by fwitte** [comment]
I will be in Berlin, too :)!

**Comment by mloenneberga** [comment]
Hi all, the next OpenMod workshop (https://www.openmod-initiative.org/) is held in Berlin in the same week. Since there are many (potential) open energy system modellers in the city, I thought it would be great to offer some kind of tutorial session or a few hours or some format for user support. What do you think?

**Comment by simnh** [comment]
> Hi all, the next OpenMod workshop (https://www.openmod-initiative.org/) is held in Berlin in the same week. Since there are many (potential) open energy system modellers in the city, I thought it would be great to offer some kind of tutorial session or a few hours or some format for user support. What do you think?

I like the idea!

**Comment by srhbrnds** [comment]
> 
> 
> > Hi all, the next OpenMod workshop (https://www.openmod-initiative.org/) is held in Berlin in the same week. Since there are many (potential) open energy system modellers in the city, I thought it would be great to offer some kind of tutorial session or a few hours or some format for user support. What do you think?
> 
> I like the idea!

Cool! If all agree, I will keep this in mind during the preparation of the openmod and bring the organization/discussion up again once the date and a rough agenda is set.

**Comment by c-moeller** [comment]
Today we roughly discussed the organisation of the coming meeting and thought, it would be good to have a news on the website as soon as possible. So, I will announce the official invitation next week. If you have anything which definitely should be mentioned within the news please comment here and I will consider this.

**Comment by busiing** [comment]
The new tab "modells" (#41 ) could be announced as: Coming soon!
With a "Call for Modells" as content for the new tab!

**Comment by c-moeller** [comment]
Sounds great :slightly_smiling_face:

**Comment by Ludee** [comment]
> > > Hi all, the next OpenMod workshop (https://www.openmod-initiative.org/) is held in Berlin in the same week. Since there are many (potential) open energy system modellers in the city, I thought it would be great to offer some kind of tutorial session or a few hours or some format for user support. What do you think?
> > 
> > 
> > I like the idea!
> 
> Cool! If all agree, I will keep this in mind during the preparation of the openmod and bring the organization/discussion up again once the date and a rough agenda is set.

FYI, the openmod has been postponed:

> After some back and forth with the logistics we would like to announce that the next Open Energy Modelling Workshop will take place from January 15 to 17, 2020 at the Hertie School in Berlin and will be hosted by the Hertie School, TU Berlin and Reiner Lemoine Institute (RLI).

**Comment by jakob-wo** [comment]
We have opened a [Wiki](https://github.com/oemof/organisation/wiki/Meeting-December-2019) to collect information about decembers dev meeting.

**Comment by c-moeller** [comment]
1) To make the wiki more attractive for interested participants, we would like to ask you to write your name to the participants list (if you already know for sure you will participate) and to add topics you like to discuss: https://github.com/oemof/organisation/wiki/Meeting-December-2019
For those who don't have write access, please comment here. We will put you on the agenda.

2) To finalize the agenda (and a general program, especially with regard to the 5th birthday of oemof) we would like to invite everybody who wants to be involved to a video or telephone conference. Here is a survey with two possible days: https://dudle.inf.tu-dresden.de/mxN07JImcw/
Right after this, we would like to announce the official agenda. Hope these days fit for you.

**Comment by jnnr** [comment]
**Do we want a poster session at the next dev-meeting?**
Last dev-meeting in Oldenburg this spring, we had a poster session in one of the afternoons. I think it was a nice option for people to introduce their use cases of oemof. What are your opinions? Would you like to have such a slot again this winter?

I think it is a good way to get into talking about applications of oemof and possible challenges one finds. Maybe one can say that it introduces a more "conference-style" feel as opposed to the more informal "hackathon-style" flavor. Don't get me wrong - I do not think that these two are in conflict. 

However, we should decide: Shall we take time for discussing posters and use this as a means to get to know each others work. Or are there other things that we should put our attention to, with the option to have another poster session at the next user meeting in 2020?

To sum up: I think that a poster session is not a must. However, if there are enough people that want a poster session and are willing to bring a poster, we should definitely have one. 

What do you think?

**Comment by FranziPl** [comment]
I want to repeat the invitation to the video or telephone conference to finalize the agenda. Please fill in the dudle, which time is convenient for you: [https://dudle.inf.tu-dresden.de/mxN07JImcw/].
We want to talk about the topics for the agenda. So, if you want a topic on the agenda, feel free to participate, even if you don't want to organize it.

**Comment by busiing** [comment]
@jnnr I liked the poster session! Repeat if enough relevant posters are contributed!

**Comment by jakob-wo** [comment]
The Dudle has spoken: We will have the WebConf on **Monday the 28th of October at 15h**
(That is next monday!)

There we will sort and cluster the topics for the dev meeting.

**Comment by jokochems** [comment]
Hi there,

so far, I have not really been contributing to oemof but I'm planning to do so.
We are currently setting up a dispatch + invest power market model for Germany (and it's electrical neighbours) using oemof at TU Berlin and I thought, it might be a good idea to get in touch for further development. For my PhD thesis, I am focussing on DSM potentials, so I probably will be using / extending the custom DSM component for this purpose.
I could give an overview over our approaches, experiences and plans and point out potential aspects for further development in a presentation. I could prepare a poster as well. I'll add myself as participant on the wiki page.
I'm sorry that my response comes quite late and would like to participate in the web conference this Monday. Are there already some dial-in details for it?

Best regards
Johannes Kochems

**Comment by uvchik** [comment]
@jokochems We are working on a dispatch/invest model for Germany as well. You may have a look here: https://github.com/reegis/deflex . At the moment we are working in improving the code and the documentation.

**Comment by jakob-wo** [comment]
**Information on todays WebConf**

Start 15:00h
Join WebConf: https://meet.jit.si/devmeeting2019
Write and document: https://etherpad.net/p/devmeeting2019_agendaprep

See you later!

**Comment by jakob-wo** [comment]
@jokochems: I added you to the participants list.

**Comment by jokochems** [comment]
> 
> 
> @jokochems We are working on a dispatch/invest model for Germany as well. You may have a look here: https://github.com/reegis/deflex . At the moment we are working in improving the code and the documentation.

Thanks a lot, I'll have a look at it.

**Comment by FranziPl** [comment]
We drew up an agenda for the meeting. Please have a look at it, especially on the parallel sessions and comment, if you have suggestions, improvements or if the parallel sessions don't work out, because someone is needed in two topics at the same time.

**Comment by FranziPl** [comment]
As you can see in the agenda, we plan a poster session. Thereby we want to give an overview of the projects and topics people are working with and make it easier to find the right person to talk to regarding a specific topic.
That's why **we want to ask you to bring a poster**, to present your project. If you don't have one and don't want to prepare one, a really simple form with a few bullet points will be fine as well.

**Comment by birgits** [comment]
Thanks for preparing an agenda! Who is planning on presenting his/her work on time series aggregation? A student of mine is currently working on that as well so maybe we could exchange on that before the meeting?

**Comment by uvchik** [comment]
Thank you!

I would like to swap 'oemof.db' and 'unbundle oemof and solph', because the 'oemof.db' discussion depend on the unbundled structure.

We do have about two hours for 'licence', poster and coffee break. Licence can be shorter, poster session and coffee break could meld a little and we would have 20-30 minutes for a "Report from the packages: tespy, feedinlib, windpowerlib, demandlib, tabular...". Could be short if there are specific session later on.

**Comment by jakob-wo** [comment]
@birgits The time series aggregation came up at the WebConf and I offered to bring people together working on that topic. I will get in contact with you (email).

**Comment by jakob-wo** [comment]
Hi @uvchik,
thanks for the feedback. 

* I swapped 'oemof.db' and 'unbundle oemof and solph' in the agenda
* I support the idea of a _Report from the packages_. It will give everyone a good overview about the oemof "cosmos" at the very beginning of the meeting. We planed everything rather tight on Wednesday to keep the evening free for the _Looking Back on 5 Yrs_-panel. On the WebConf @gplssm said he will get in touch with you to discuss that panel. As soon as you two know how much time is needed we could also adapt the schedule.

**Comment by FranziPl** [comment]
As it was not clear in my last comment: Please bring a poster on paper, not a file of a poster.

**Comment by gplssm** [comment]
> @birgits The time series aggregation came up at the WebConf and I offered to bring people together working on that topic. I will get in contact with you (email).

It's on my task list to write you a mail about that. I'll do it soon

**Comment by uvchik** [comment]
> As it was not clear in my last comment: Please bring a poster on paper, not a file of a poster.

I was talking about the content not the media. At the user meeting people just brought their project poster but I guess that most people do not have a poster about developing topics. At least we do not have one. What was your idea about that.

**Comment by FranziPl** [comment]
@uvchik sorry, I didn't read your question, someone else was asking via mail.
@jnnr is in charge for the poster session, so he will answer you.

**Comment by jokochems** [comment]
> 
> 
> We drew up an agenda for the meeting. Please have a look at it, especially on the parallel sessions and comment until **friday, the 8th of November**, if you have suggestions, improvements or if the parallel sessions don't work out, because someone is needed in two topics at the same time.

@FranziPl: Thanks a lot, it looks quite good to me.

> 
> 
> @birgits The time series aggregation came up at the WebConf and I offered to bring people together working on that topic. I will get in contact with you (email).

@jakob-wo: As we discussed in the meeting, I've got a student of mine working on that, too, who is interested in getting involved in that session.

> **#poster** If you want to bring a poster showing your research projects using oemof, please inscribe your name in this [list](https://github.com/oemof/organisation/wiki/Meeting-December-2019#posters) beneath the agenda. If you do not have writing permissions on the wiki, let me know and I will do it for you.

@jnnr: I'll prepare a poster showing our general modelling approaches for power market modelling and adressing some aspects for further development (from a usage & development perspective); working title: "Modelling issues around the ER fundamental power market model" ...We're still looking for a more catchy name.

**Comment by jnnr** [comment]
> @jnnr: I'll prepare a poster showing our general modelling approaches for power market modelling and adressing some aspects for further development (from a usage & development perspective); working title: "Modelling issues around the ER fundamental power market model" ...We're still looking for a more catchy name.

Sounds good. I put it on the agenda. From what you write it seems to fit a dev meeting focused on developer topics and new approaches, which is nice.

**Comment by birgits** [comment]
@gnn and I would like to add a parallel session about the feedinlib, preferably on Friday, to present the current status and plan and discuss further developments.

**Comment by fwitte** [comment]
On thursday afternoon there was a TESPy heat components session. I would like to prepare a coupled model integrating tespy and solph and thus renamed the session. Please let me know, if there are any reservations against this name change.

**Comment by jnnr** [comment]
I would like to include a session on district heating networks. It was listed as a topic on the telco but somehow dropped from the current schedule. Can we find a good place for it?

**Comment by c-moeller** [comment]
I added the topic of @birgits (Friday afternoon) and @jnnr (Wednesday afternoon) and changed the time for the birthday celebration. There are now three more timeslots in the afternoon of Wednesday.

**Comment by uvchik** [comment]
As mentioned above I would add a plenum session "Brief report from the packages" (feedinlib, windpowerlib, tespy, tabular):

- Whats new
- What is planned
- Problems
- Experience with integrated Tools

We could add a short discussion about integrated tools at the end, what tools are recommended or not recommended and what is the best way to integrate them (e.g. flakes8, lgtm, pep8speaks....).

**Comment by uvchik** [comment]
Furthermore, we could have a look at issues/PRs that got stuck because of missing feedback. We could 'force' the feedback in a short debate.

**Comment by fwitte** [comment]
> As mentioned above I would add a plenum session "Brief report from the packages" (feedinlib, windpowerlib, tespy, tabular):
> 
>     * Whats new
> 
>     * What is planned
> 
>     * Problems
> 
>     * Experience with integrated Tools
> 
> 
> We could add a short discussion about integrated tools at the end, what tools are recommended or not recommended and what is the best way to integrate them (e.g. flakes8, lgtm, pep8speaks....).

Very good idea!

**Comment by joroeder** [comment]
> 
> 
> Furthermore, we could have a look at issues/PRs that got stuck because of missing feedback. We could 'force' the feedback in a short debate.

I like that idea! ;)

**Comment by c-moeller** [comment]
@uvchik I added the report from the packages at Wednesday afternoon.

> Furthermore, we could have a look at issues/PRs that got stuck because of missing feedback. We could 'force' the feedback in a short debate.

@uvchik Isn't this topic still covered with the plenary on Thursday afternoon (13:45)?

I opened a pad and would like to transfer the "detailed description of topics" below the agenda to the pad and adapt it a bit. Then everybody has write access and can describe his or her agenda point. Any objectives? Please write here. Otherwise I will do it tomorrow :slightly_smiling_face:

**Comment by c-moeller** [comment]
Is there interest in insights on efficient GitHub workflows? @jnnr and I had a workshop some days ago and we could pass our new knowledge :slightly_smiling_face:

**Comment by uvchik** [comment]
> Is there interest in insights on efficient GitHub workflows? @jnnr and I had a workshop some days ago and we could pass our new knowledge slightly_smiling_face

:+1:

**Comment by ckaldemeyer** [comment]
> Is there interest in insights on efficient GitHub workflows? @jnnr and I had a workshop some days ago and we could pass our new knowledge slightly_smiling_face

Definitely!

**Comment by c-moeller** [comment]
> Definitely!

Great! I added a session at Thursday afternoon.

I now opened a pad, where we can describe our sessions with a short teaser text (no need for complex texts, one to three sentences are enough): https://etherpad.wikimedia.org/p/oemof_dev_meeting_dec_2019

@oemof-developer Please add a teaser text for your session(s). I copied the notes from the wiki, but I think it's better to have a nice short text. I would ask you to do that as soon as possible, since I'd like to announce the final agenda at oemof.org. Thank you!! :slightly_smiling_face:

**Comment by c-moeller** [comment]
There is now some space for additional topics on Wednesday afternoon. I think it's not so good to leave this space in the beginning of the meeting. But now some people already planned their attending days.

@oemof-developer Do you see any chance to shift some sessions to Wednesday?

**Comment by birgits** [comment]
Either the windpowerlib or feedinlib session could be moved to Wednesday as far as I'm concerned.

**Comment by jakob-wo** [comment]
Before moving the windpowerlib session please contact @SabineHaas, the Wiki says she was not sure whether she will attend on Wednesday or only on Thursday. That is why I placed the windpowerlib on Thursday in the first place.

**Comment by gplssm** [comment]
I started to prepare some slides. Thereby, the question arose which language I should. In a quick discussion with @jnnr and @c-moeller, we decided to prepare all slides in English.
If participants in a session are all German speakers, we can talk German in the session.

**Comment by SabineHaas** [comment]
> Either the windpowerlib or feedinlib session could be moved to Wednesday as far as I'm concerned.

I'd prefer to stick with the Thursday's windpowerlib session.

**Comment by jnnr** [comment]
We want to remind you about the poster session on the upcoming oemof-dev-meeting in December.

In this poster session you can present your current work and have the chance to connect with other
developers in your scientific field. As the winter meeting has a developer focus, we would like
the contributions to address your ongoing developments, interesting new methods or approaches worth discussing.

There are no formal requirements. If you want to show your problems and approaches, feel free
to bring a collection of A4/A3 pages and pin them on the board so we can discuss. We prefer an interesting exchange to shiny perfectionism, so a more improvised version would count as a poster as well.

If you want to use this opportunity, please bring a poster! Let me know (just post in this issue), so I can put your name and the title of your poster to the agenda.

**Comment by c-moeller** [comment]
> I'd prefer to stick with the Thursday's windpowerlib session.

Ok! I only shifted the feedinlib session to Wednesday.

@oemof-developer Anyone else is able to shift his or her session?

**Comment by c-moeller** [comment]
> @oemof-developer Anyone else is able to shift his or her session?

@busiing What about the models tab/website session?

**Comment by jakob-wo** [comment]
> @jakob-wo: As we discussed in the meeting, I've got a student of mine working on that, too, who is interested in getting involved in that session.

@jokochems: I talked to @sbruche and @birgits about the tsam-Session. Both are going to prepaire a 10-15min input for the session. I think if your student would prepaire for 10-15min as well these three inputs will be a solid base of discussion and exchange.

**Comment by c-moeller** [comment]
The final news post is now online: https://oemof.org/2019/11/21/anniversary-developer-meeting-final-agenda/

Registration is open until November 27th.

**Comment by c-moeller** [comment]
> @busiing What about the models tab/website session?

After consulting @busiing I shifted the models tab session to Wednesday.

I also added a demandlib session on Wednesday and an optional session on Thursday to bring the website up to date (can be skipped)

**Comment by SabineHaas** [comment]
@uvchik Should we prepare anything for the session "Report from the packages" on the first day? What's your plan for this time slot?

**Comment by joroeder** [comment]
I would like to attract notice to [#594](https://github.com/oemof/oemof/issues/594). I would appreciate discussing this topic within a plenary session (maybe within the 'Pending Issues' session on Thursday). There is the 'facade' topic in the parallel session before lunch, which could do some summary of what's been discussed (actually Jan did this already in #594). This would be a great opportunity to come to an agreement for this again and again upcoming topic. In my opinion, some agreement on that topic is very helpful for further developments of many kinds of oemof-solve applications.

**Comment by uvchik** [comment]
> @uvchik Should we prepare anything for the session "Report from the packages" on the first day? What's your plan for this time slot?

I just saw that I am in charge for this session.

I would ask each oemof package for a small presentation (max. 5 minutes)

* windpowerlib @SabineHaas 
* tespy @fwitte 
* oemof-thermal @?
* feedinlib @gnn, @birgits 
* oemof-tabular @gnn 
* cydets @?
* demandlib @uvchik 
* solph @?
* oemof_db @uvchik, @gplssm 

The topic licence will not need 20 minutes so we will have about 40 minutes.

Please sent me your slides in pdf format that we do not have to change computers between the short talks.

**Comment by jokochems** [comment]
> 
> 
> > @jakob-wo: As we discussed in the meeting, I've got a student of mine working on that, too, who is interested in getting involved in that session.
> 
> @jokochems: I talked to @sbruche and @birgits about the tsam-Session. Both are going to prepaire a 10-15min input for the session. I think if your student would prepaire for 10-15min as well these three inputs will be a solid base of discussion and exchange.

@jakob-wo: Sorry that my answer took quite a while: That's fine: A student of mine will prepare a short presentation and share his experience with tsam. Since so far, he has been working on getting an overview over all features and methods included as well as testing them in smaller model settings, his presentation could be placed at the beginning of the session before detailed discussions and more complicated use cases ensue.
@c-moeller: Would you please add him to the participants list (Hannes Kachel, TU Berlin)(only for the tsam session)? Thanks a lot!

**Comment by c-moeller** [comment]
@c-moeller: Would you please add him to the participants list (Hannes Kachel, TU Berlin)(only for the tsam session)? Thanks a lot!

> done :+1:

**Comment by c-moeller** [comment]
I extended one coffee break on Thursday to have a bit more space and changed the coffee break on Friday to a "Laboratory and Coffee Break" --> interested people will have the chance to visit the great laboratory of the Beuth Hochschule :-)

**Comment by c-moeller** [comment]
> I would like to attract notice to #594. I would appreciate discussing this topic within a plenary session (maybe within the 'Pending Issues' session on Thursday). There is the 'facade' topic in the parallel session before lunch, which could do some summary of what's been discussed (actually Jan did this already in #594). This would be a great opportunity to come to an agreement for this again and again upcoming topic. In my opinion, some agreement on that topic is very helpful for further developments of many kinds of oemof-solve applications.

Good point :+1: We could add that to the plenary session on Thursday from 15:30. Will there be enough time? And until now nobody is in charge for the facades parallel session. @uvchik and @jakob-wo who are assignees of issue #594 are in other parallel sessions. @oemof-developer anyone else who can lead the facades session?

**Comment by c-moeller** [comment]
Also the oemof foundation plenary discussion is looking for a leader. @uvchik @busiing #44 ? Or any other ideas?

**Comment by uvchik** [comment]
At the last meeting we had a discussion about errors/checks/warnings, that make it easier to find bugs in your application. Nothing has happened for 6 month but I have an idea for a compromise. It is a petty to miss the other sessions but I could offer a DO on Friday.

The problem is that solph is very generic and a lot of things are allowed but unwanted in most but not all cases. For most users some more errors/warnings/etc. would help to find errors/typos/etc. but for experienced users might be annoyed by "wrong" warnings or even hampered by errors.

We could work out a compromise in this session.

**Comment by busiing** [comment]
I could lead the session, but I feel a little insecure about the lead for
that discussion. The issue is pretty clear, that we still need responses
from the major institutions, who created oemof in the first place. (RLI,
Flensburg)

In addition to that, we need 7 people to create a legal oemof foundation.

Am Mo., 2. Dez. 2019 um 11:21 Uhr schrieb Caroline Möller <
notifications@github.com>:

**Comment by joroeder** [comment]
I could do the lead of the facade discussion session. (Thursday 11:00 am - d) )

**Comment by joroeder** [comment]
If there is some time in the poster session left, I could show two slides of district heating network optimization with oemof.solph ( I don't have a poster)

**Comment by busiing** [comment]
Bin gerade bisschen lost in Haus A. Si miss ich hin?

Johannes Röder <notifications@github.com> schrieb am Mi., 4. Dez. 2019,
11:32:

**Comment by uvchik** [comment]
Thanks again to the organisation team. It was great and well organised meeting :tada: And the best location ever :smile:

**Comment by jakob-wo** [comment]
I transfered the protocol from the pad to the [Wiki](https://github.com/oemof/organisation/wiki/Meeting-December-2019).
Please have a look if everthing important was written down and feel free to add missing information.

**Comment by fwitte** [comment]
I will close this, feel free to reopen if required :).
